scripts/chroot_reconst.sh

```diff
@@ -58,37 +58,52 @@ rm  /usr/share/xsessions/plasma.desktop
 # ugliest hack ever
 cp  /usr/share/xsessions/lxqt.desktop /usr/share/xsessions/plasma.desktop
 
-# Installing 
-apt-get -qq -y install git
+# 3rd party software installation
+if ["chromium" in "$INSTALL"]
+then
+	apt-get install -y chromium-browser
+else if ["firefox" in "$INSTALL"]
+then
+	apt-get -qq -y install firefox
+else if ["chrome" in "$INSTALL"]
+then
+	if [[ $(arch) = "amd64" ]]
+	then
+		wget -q https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb &&
+		dpkg -i google-chrome-stable_current_amd64.deb &&
+		rm -f google-chrome-stable_current_amd64.deb
+	else
+		wget -q https://dl.google.com/linux/direct/google-chrome-stable_current_i386.deb &&
+		dpkg -i google-chrome-stable_current_i386.deb &&
+		rm -f google-chrome-stable_current_i386.deb
+	fi
+else if ["vlc" in "$INSTALL"]
```

Comment:
What if both vlc and git are in install only vlc will be installed. Maybe, try replacing else if s with if s

Reply:
yep, there should not be any else if part. Thanks :)